index.html

```diff
@@ -12,6 +12,8 @@ <h1>Celine Vogel</h1>
     <nav>
         <a href="#contact">Contact</a>
         <a href="#biography">Biography</a>
+        <a href="#socials">Socials</a>
+        <a href="#cv">CV</a>
```

Comment:
I think Biography should cover this point as well. But I might be wrong or you have other inspirational ideas :-)